mountpoint-s3/src/cli.rs

```diff
@@ -639,15 +638,21 @@ where
 
     let prefetcher_config = Default::default();
 
-    if let Some(path) = args.cache {
-        let metadata_cache_ttl = args.metadata_ttl.unwrap_or(Duration::from_secs(1));
-        filesystem_config.cache_config = CacheConfig {
-            serve_lookup_from_cache: true,
-            dir_ttl: metadata_cache_ttl,
-            file_ttl: metadata_cache_ttl,
-            ..Default::default()
-        };
+    let mut metadata_cache_ttl = args.metadata_ttl.unwrap_or_else(|| {
+        if args.cache.is_some() {
+            // When the data cache is enabled, use 1min as metadata-ttl.
+            TimeToLive::Custom(Duration::from_secs(60))
+        } else {
+            TimeToLive::Minimal
+        }
+    });
+    if matches!(metadata_cache_ttl, TimeToLive::Custom(Duration::ZERO)) {
+        tracing::warn!("'--metadata-ttl 0' is not supported, using 'minimal' instead");
```

Comment:
We need to make clear that we're planning to remove this in a future release. So two things:
1. Wording needs to be more direct: something like `The '--metadata-ttl 0' setting is no longer supported, is now interpreted as 'minimal', and will be removed in a future release. Use '--metadata-ttl minimal' instead.`
2. We might want to send this message somewhere other than tracing -- like making sure it gets printed to stderr even when mounting in background mode.

Reply:
1. done
2. agree, but not trivial at first glance. I'll think about it a bit more.